phpBB/phpbb/di/extension/core.php

```diff
@@ -71,7 +71,7 @@ public function load(array $configs, ContainerBuilder $container)
 
 		// Set the Twig options if defined in the environment
 		$definition = $container->getDefinition('template.twig.environment');
-		$twig_environment_options = $definition->getArgument(7);
+		$twig_environment_options = $definition->getArgument(6);
```

Comment:
Can we please define a constant for this? Magic numbers shouldn't be in newer code.